### DISCUSSION TITLE
Unable to post a discussion

### DISCUSSION THREAD

**Discussion by sryze** [community/community#13861] [Discussions]
The "Start discussion" button remains disabled after you fill the title and post content. It's not obvious (at least for me) that you need to choose a category from the dropdown, as there are no hints being  displayed about that.

Please highlight it as required or add some kind of tooltip next to the submit button.

Thanks.

**Comment by chalemen** [comment]
Todo lo ise por medio de un solo teléfono no use laptop ni table ni algo parecido mañana si gustan alas 14:00 horas me conectaré y veré si en verdad han ayudado a mi México querido mi país ya ucrania y alos que fueron afectados me da gusto que se parara laguerra!!Pero si algún país kiere o anuncia otra guerra no solo le quitaré su capital si no que dejaré limpia su tarjeta madre con millas de archivos para su país e infección a sus apps para que le sea difícil pararse y no me gaste nada de dinero de finanzas lo tengo todo en mi cuenta por si gustan mañana se los daré para apoyar alas personas con pocos recursos y lo siento por tantos problemas pero ya me canse de ver muertes a familias con un gran futuro por delante arruinado solo por querer mas poder eso no va y mucho gusto me llamo Carlos y soy de guadalajara jalisco y mañana doy mis calles y número de casa por si gusta pasar por mi para aveŕ si me contrata alguien para ayudar al futuro del mundo para ser mejor....yo tengo pocos recursos trabajo de guardia de seguridad orita jajajaja pero siempre e visto como crecer y espero ver al mundo crecer con migo y aki tienen todo mi apoyo aun que no sea mucho lo poco que tengo es tuyo y mio!¡!!
